internal/impl/pulsar/output.go

```diff
@@ -14,33 +14,7 @@ import (
 func init() {
 	err := service.RegisterOutput(
 		"pulsar",
-		service.NewConfigSpec().
-			Version("3.43.0").
-			Categories("Services").
-			Summary("Write messages to an Apache Pulsar server.").
-			Field(service.NewURLField("url").
-				Description("A URL to connect to.").
-				Example("pulsar://localhost:6650").
-				Example("pulsar://pulsar.us-west.example.com:6650").
-				Example("pulsar+ssl://pulsar.us-west.example.com:6651")).
-			Field(service.NewStringField("topic").
-				Description("The topic to publish to.")).
-			Field(service.NewObjectField("tls",
-				service.NewStringField("root_cas_file").
-					Description("An optional path of a root certificate authority file to use. This is a file, often with a .pem extension, containing a certificate chain from the parent trusted root certificate, to possible intermediate signing certificates, to the host certificate.").
-					Default("").
-					Example("./root_cas.pem")).
-				Description("Specify the path to a custom CA certificate to trust broker TLS service.")).
-			Field(service.NewInterpolatedStringField("key").
-				Description("The key to publish messages with.").
-				Default("")).
-			Field(service.NewInterpolatedStringField("ordering_key").
-				Description("The ordering key to publish messages with.").
-				Default("")).
-			Field(service.NewIntField("max_in_flight").
-				Description("The maximum number of messages to have in flight at a given time. Increase this to improve throughput.").
-				Default(64)).
-			Field(authField()),
+		pulsarSepc(),
```

Comment:
extracted in order to add UT. I didn't check-in UT because it is only partially done